.github/workflows/snap.yaml

```diff
@@ -16,11 +16,16 @@ on:
 
 jobs:
   snap:
-    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@main
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: ['["ubuntu-latest"]', '["ARM64", "self-hosted", "linux", "large", "noble"]']
+    uses: canonical/robotics-actions-workflows/.github/workflows/snap-lxc.yaml@fix/snap-lxc-inputs-mimic
```

Comment:
This seems to be a temporary branch?

Reply:
Yes this is relying on the non merged branch: https://github.com/canonical/robotics-actions-workflows/pull/3